packages/webapp/src/members/api/eden-contract.ts

```diff
@@ -10,10 +10,15 @@ import {
 import { EdenMember, MemberStats } from "../interfaces";
 import { TreasuryStats } from "../../pages/api/interfaces";
 import { devUseFixtureData } from "config";
-import { fixtureMembersStats } from "delegates/api/fixtures";
+import { fixtureEdenMembers, fixtureMembersStats } from "./fixtures";
 
-export const getEdenMember = (account: string) =>
+export const getEdenMember = (account: string) => {
+    if (devUseFixtureData)
+        return Promise.resolve(
+            fixtureEdenMembers.find((member) => member.account === account)
+        );
```

Comment:
Cool! I did something very similar in my draft, the difference is that I'm falling back to a regular table call if the fixture is not found: https://github.com/eoscommunity/Eden/pull/361/files#diff-b5c705d489c324900aeac335703bf0ea500c8f433a0e0a328a640c22e2f88303R15-R25

Should we consider that? In any case you can keep like this and then I could merge mine over it if you think that's interesting.

Reply:
@sparkplug0025 This does fall back to a regular table call. That's the structure of everything I've written in this PR. The next line, L20, has the table call.

Reply:
Actually it does not, if the fixture flag is on and the fixture data is not present you are returning undefined. (Which is totally fine, I was wondering if the fixture is not present we could still do a table call 😊)

Reply:
added in https://github.com/eoscommunity/Eden/pull/370